_pages/team.md

```diff
@@ -1,16 +1,58 @@
+
+In a career of over two decades he conceptualized and built internationally awarded telemedicine devices ( HMSEI / RDK / Dr Rho ), blended reality autonomous vehicles ( TARA ) and cognitive robots ( ANGEL ).He has also worked for Hewlett Packard as a platform architect for software defined networking, a Cummin franchise on industrial automation and intelligent vehicles and a UK based fintech provider on intelligent regulatory reporting. Accomplished innovator with multiple patent filings and publications, he has been a pioneer in inclusive technologies having developed Hindawi Programming System for systems programming in Hindi and other Indian languages.
+
+He is multidisciplinary trained in Pre-clinical Medicine, Neural Engineering, Medical Devices Technology, Telemedicine, Remote Diagnostics, Augmented Reality, Robotics & AI (Natural Language Processing, Machine Learning, Deep Learning, Analytics and Data Sciences).Qualifications also include MBA, Law, Clinical Psychology, Linguistics, Philosophy and Genetic Anthropology. He started as a freelancer in 1996 working on embedded systems and has been an Entrepreneur between 2000 - 2006 building medical devices and autonomous robots. His research areas are Brain Computer Interface and Brain Implants (electro-chemical implants for Parkinson's disease, Alzheimer's disease and developmental cognitive disorders), Technological Singularity and Transhumanism.
+
+## &#10070;
+
+<img src="https://projectvikram.github.io/restricted_assets/images/dr_srija.jpg" alttext="Dr Srija Katta (avatar image)" width="300" height="300">
+
+## Dr Srija Katta 
+### Technology Evangelist-Pharma and Convenor
+#### @srija-ieee
+
+Dr Srija is the Co-Founder of AyeAI Consulting and Heads Company Operations and Pharma lab. She is a Director at Vyas Labs Pvt Ltd and Interglial Healthcare (OPC) Pvt Ltd. 
+
+She is a passionate Entrepreneur experienced in Startup strategy, New Product Road Map, Requirement Analysis, with skills in Artificial Intelligence for Pharma and Healthcare. She was one among the invitees of Roundtable “Transforming Primary Healthcare – Leveraging Technologies” at NASSCOM Technology and Leadership Forum 2019. She was an invited speaker for IEEE CIS Summer School 2018 and has delivered a handson session on topic “AI in Healthcare” Dr Srija is familiar with Course design, Seminars and Workshop delivery. 
+
+Dr Srija is a Clinical Pharmacist (Doctor of Pharmacy) by profession, trained in Biz ops like Regulatory Compliance, Banking Operations, HR management, Public Relations, Business development Collaterals design, Awareness and Marketing Campaign design and other technical aspects like Gitops, Website building etc.. She is skilled in Systemic and Conversational Peer Review and has been a Reviewer for a few international conferences and journals including International Society of Pharmacoeconomics and Outcomes Research 6 TH Latin America Conference, Brazil - 2017 and 20th Annual European Congress.-2017.
```

Comment:
"Biz ops" -> "biz-ops"

Reply:
HR Management -> M capital
Business Dev -> D capital / comma after Development
Design -> D capital
Gitops -> GitOps
Website building -> Website development
etc.. -> Single full stop
Instead of etc.., say - among others.
After ISPOR add comma
6 TH -> 6th